### DISCUSSION TITLE
How should excute command "lotus chain prune"?

### DISCUSSION THREAD

**Discussion by lbj2004032** [filecoin-project/lotus#9574] [Storage Provider] [answered]
Every few months, the disk of lotus fullnode will be full. We need to stop the lotus process, which leads to computing power loss. I see that v1.17.1 has the function of automatic pruning (lotus chain prune), but no documents are used. How do you want to achieve pruning without restarting?

**Comment by rjan90** [comment, marked as answer]
Hey @lbj2004032!

The SplitStore is documented here: https://lotus.filecoin.io/lotus/configure/splitstore/ - you can also watch the showcase video of the feature here: https://www.youtube.com/watch?v=eBWdTJgUPIU&t.

You will be able to enable the AutoPrune feature in SplitStore once this issue gets resolved: https://github.com/filecoin-project/lotus/issues/9236. It´s on the agenda to fix this issue in Q4 this year.
